### PR TITLE
V4.9.0

### PR DESCRIPTION
**Pull request by kagg-design**

### CODE REVIEW

**Comment by kagg-design** [comment]
@e271828- Could you please review and merge?

File: .tests/php/integration/Admin/NotificationsTest.php [hunk 1]
```diff
@@ -168,6 +168,30 @@ public function test_get_notifications( bool $empty_keys, bool $pro, bool $force
...
+				],
+			],
+			'post-vs-turnstile'   => [
+				'title'   => 'hCaptcha vs. Turnstile',
+				'message' => 'How does Turnstile compare to hCaptcha?',
+				'button'  => [
+					'url'  => 'https://www.hcaptcha.com/post/hcaptcha-vs-turnstile/?r=wp&utm_source=wordpress&utm_medium=wpplugin&utm_campaign=not',
+					'text' => 'Compare',
+				],
+			],
```
Comment (e271828-) [inline .tests/php/integration/Admin/NotificationsTest.php:194]:
```suggestion
```

Appreciate the thought, but we don't need to link to blog posts here. Rather keep it focused on Pro.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,20 @@
+= 4.9.0 =
+* Added LearnPress integration.
+* Added Tutor LMS integration.
+* Added compatibility with Ninja Forms v3.8.22.
+* Added the ability to install plugins and themes from the Integrations page.
+* Added comparison with reCAPTCHA and Turnstile to the Notifications sections on the General page.
```
Comment (e271828-) [inline changelog.txt:6]:
```suggestion
```

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+* Added ability to hide the login errors.
+* Added anonymous collection of IP and User Agent data to conform to GDPR requirements.
```
Comment (e271828-) [inline changelog.txt:8]:
```suggestion
* Added anonymous collection of IP and User Agent data in locally stored analytics to simplify GDPR compliance.
```

File: readme.txt [hunk 4]
```diff
@@ -610,6 +620,23 @@ Instructions for popular native integrations are below:
...
 == Changelog ==
 
+= 4.9.0 =
+* Added LearnPress integration.
+* Added Tutor LMS integration.
+* Added compatibility with Ninja Forms v3.8.22.
+* Added the ability to install plugins and themes from the Integrations page.
+* Added comparison with reCAPTCHA and Turnstile to the Notifications sections on the General page.
+* Added ability to hide the login errors.
+* Added anonymous collection of IP and User Agent data to conform to GDPR requirements.
```
Comment (e271828-) [inline readme.txt:630]:
```suggestion
* Added anonymous collection of IP and User Agent data in locally stored analytics to simplify GDPR compliance.
```

**Comment by kagg-design** [comment]
@e271828- Thank you for the suggestions; I have gone ahead with them.

**Review by e271828- — APPROVED** [review]

**Pull request merged**